modules/webgl/src/context/context.js

```diff
@@ -108,6 +108,9 @@ export function instrumentGLContext(gl, options = {}) {
 
   gl._version = gl._version || getVersion(gl);
 
+  // Cache canvas size information to avoid setting it on every frame.
+  gl._canvasSizeInfo = {};
```

Comment:
How about adding one object valued propsto the `gl` context and then additional info as subfields, rather than a bunch of props? I believe we already add a `gl.luma` object that could be used.

`gl.luma.canvasSize` ?

Reply:
Yep, that sounds good, will update.